src/stylesheet/Quiz.css

```diff
@@ -0,0 +1,107 @@
+.QuizDetail{
```

Comment:
please take care of spacing.
Also convention is to use hyphenated class names instead of Camel case names

Reply:
Done